ddmrp_adjustment/models/stock_warehouse_orderpoint.py

```diff
@@ -39,7 +38,7 @@ def _calc_adu(self):
             for val in values:
                 daf *= val
             prev = self.adu
-            self.adu *= daf
+            self.with_context(__calc_adu=True).adu *= daf
```

Comment:
where is this context used?

Reply:
@lreficent We did change the context key after the review here : https://github.com/Eficent/ddmrp/pull/47/commits/578389f8729577604c3eed522344a99a3eddf399
Just added a commit to fix it.

Reply:
oh, I see that, TBH I don't like it, not here, I don't like the fact to force the ADU calculation on the write method of the orderpoints. ADU computation MUST rely on the cronjob. But no worries I will have to disccus with @jbeficent  and will probably add an extra commit to remove that (if needed) once everything is merged.